app/Http/Controllers/SubCategoryController.php

```diff
@@ -148,4 +158,19 @@ public function destroy($id)
         Session::flash('message', 'Sub Category Deleted Successfully.');
         return redirect('admin/categories');
     }
+    public function changeSubCategoryStatus(Request $request)
```

Comment:
I am seeing the same code in three different controllers, can we please get rid of duplicated code? Maybe include a module for this?